x-pack/plugins/reporting/server/routes/lib/request_handler.test.ts

```diff
@@ -110,7 +110,7 @@ describe('Handle request to generate', () => {
           "kibana_name": undefined,
           "max_attempts": undefined,
           "meta": Object {
-            "isDeprecated": false,
+            "isDeprecated": undefined,
```

Comment:
This is not exactly correct (should be `true` since there is a V2 type), but is related to https://github.com/elastic/kibana/issues/121365